### PR TITLE
Add Home and About Pages for Mobile and Desktop

### PR DESCRIPTION
**Pull request by frempongdev**
In this Pull Request I made the following changes: ♻️ 

-  Added Home and About Page to Project
- - Added page structure for both Moblie and Desktop versions. 
- - Added Style to Mobile Section First. 📱 
- - Added Mobile Nav by implementing Hamburger and the various nav links,
- - Added Desktop style and page images. 🖥️ 
- - Added javascript DOM dynamic elements for the speakers.
- - Implemented buttons for revealing more speakers on mobile.
- - Added standard readme.md. 
- - Fixed Linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,181 @@
...
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [👥 Authors](#authors)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
```
Comment (OLIPLICHE) [inline README.md]: - Please add this section to the table of content and make sure sections in your Readme correspond to the sections mentioned inside the table of content
- 
Capture

File: about.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                    <a href="" class="parner-link">
+                        <img src="assets/partner/sparena-logo.jpg" alt="sparena-logo">
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </section>
+
+    <!-- FOOTER--------------------------------------------------------------------------------------- -->
+    <footer class="footer">
```
Comment (OLIPLICHE) [inline about.html:152]: - [x] **Mobile Size: Home/About**
- Make sure it has a white background:
- **See your design**
Capture1
-
- **See the original**
Capture

File: about.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        <span>Take a look at the two Esports Festivals which took place in Kumasi and Tamale</span>
+        <div class="events-box">
+            <a href="#" class="event1">
+                <img src="assets/event1.png" alt="event1-img">
+            </a>
+            <a href="#" class="event1">
+                <img src="assets/event2.png" alt="event2-img">
+            </a>
+        </div>
+    </section>
```
Comment (OLIPLICHE) [inline about.html:107]: - [x] **About Page:**
- Please follow the original design: The color of the pictures should look like in the original design
- See your design:
Capture
- Original Design:
Capture1

File: index.html [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+            </button>
+            <button id="less" class="show-more hide-speaker">
+                <span>LESS</span>
+                <img class="drop-down" src="assets/arrow-up.png" alt="drop-down-arrow">
+            </button>
+        </div>
+    </section>
+
+<!-- PARTNERS SECTION ----------------------------------------------------------------------------- -->
+<section class="partners hide-class">
```
Comment (OLIPLICHE) [inline index.html:144]: - [x] **Mobile size: HOME**
- After clicking on more, I noticed that you don't have a Partner section, Please add the partner section
- 
Capture

File: index.html [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+                </div>
+            </div>
+            <div class="speaker-box">
+                <img src="assets/speakers/speaker_06.png" alt="speaker1-img" class="speaker-image">
+                <div class="speaker-desc-box">
+                    <h4 class="speaker-name">Igor</h4>
+                    <p class="speaker-title">Madagastar Esports Arena, West-Hills Mall, Weija-Accra</p>
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div> -->
```
Comment (OLIPLICHE) [inline index.html]: - Please remove these deleted code to make your work clean

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @frempongdev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linter is green (the last commit has a green check mark (✅)) 👏
- Commit messages are meaningful 👏 👏

## Required Changes ♻️

- [x] Video presentation: 
- Please you need to record again. Highlight a piece of the code you find interesting or proud of making.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 3 above)
Comment (m05e5) [inline about.html:107]: - [x] Hi. you did a great job but as mentioned by the previous reviewer kindly make sure you follow the design pattern as much as possible. So, make sure to have the same background color added to your images

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @frempongdev  , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [x] please the comments below

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 3 above)
Comment (frempongdev) [reply about.html:107]: This was an optional suggestion from the previous reviewer. is there any other reason why this project cannot pass?

**Review by codecaiine — APPROVED** [review]
Hi @frempongdev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great required changes implementation :heavy_check_mark: 
- Good use Github flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (frempongdev) [reply README.md]: Alright @OLIPLICHE, thank You for your suggestion.
All the changes have been made.

File: index.html (hunk 5 above)
Comment (frempongdev) [reply index.html]: All done.

**Comment by frempongdev** [comment]
Thank You @OLIPLICHE , @m05e5 , @codecaiine for taking a deep dive into this Pull Request. I really appreciate your reviews.

@codecaiine Thanks once again for the approval.

File: about.html (hunk 2 above)
Comment (frempongdev) [reply about.html:152]: Thanks @OLIPLICHE for this suggestion. The changes you requested have been made. 👍🏿

File: about.html (hunk 3 above)
Comment (frempongdev) [reply about.html:107]: All the changes requested have been made. Thanks @OLIPLICHE. 👌🏿

File: index.html (hunk 4 above)
Comment (frempongdev) [reply index.html:144]: Hi @OLIPLICHE ,
The partner section of this project, according to the original design is found at the about page so I placed it in that page. Kindly verify from `about.html` where the partners section is present.

File: about.html (hunk 3 above)
Comment (frempongdev) [reply about.html:107]: I have made due changes as requested by you and the previous reviewer. Thank You @m05e5. 👏🏿

**Pull request merged**